tests/load/pipeline/test_merge_disposition.py

```diff
@@ -989,3 +989,24 @@ def r():
     with pytest.raises(PipelineStepFailed) as pip_ex:
         p.run(r())
     assert isinstance(pip_ex.value.__context__, SchemaException)
+
+
+@pytest.mark.parametrize(
```

Comment:
Can you write a test (or check if one exists) to see what happens when we do a merge on merge keys but some rows have null in the merge key? It's not super important right now, but if it would be interesting to know what happens :)

Reply:
I couldn't find a test so I added one. This was raising an exception already through `schema.coerce_row` in normalize